src/iocore/net/SSLSecret.cc

```diff
@@ -35,6 +36,20 @@ DbgCtl dbg_ctl_ssl_secret_err{"ssl_secret_err"};
 
 } // end anonymous namespace
 
+static void
+get_hash_str(const std::string &input, char hash_str[EVP_MAX_MD_SIZE * 2], unsigned int *hash_len)
+{
+  unsigned char hash[EVP_MAX_MD_SIZE];
```

Comment:
Maybe initialize the arrays with `= {0};` ?

Reply:
Hmmm, I don't think that helps something.